test_resolve.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  PolymorphicResolver,
+  serializePolymorphicClass,
+} from "./polymorphic.ts";
+import { JsonValue, Serializable } from "./serializable.ts";
+import { SerializeProperty } from "./serialize_property.ts";
+
+abstract class MyBaseClass extends Serializable {
+  @SerializeProperty()
+  public parentValue?: number;
+
+  public testMethod() {
+    console.log("Default logic");
```

Comment:
can we return this as a string and use `assertEquals` also noticing now that this is not using the deno testing infra

Reply:
Added real tests